tests/reading-tests.md

```diff
@@ -436,18 +436,26 @@ Using your own words, describe one thing that you would watch for while reading
 <!-- Question Takeaway -->
 <!-- prettier-ignore-start -->
 ### !challenge
-* type: paragraph
+* type: ordering
 * id: 0c5b9845
 * title: Reading Tests
 ##### !question
 
-What was your biggest takeaway from this lesson? Feel free to answer in 1-2 sentences, draw a picture and describe it, or write a poem, an analogy, or a story.
+Arrange the lines of code below in the following order so that they create a Jest test:
```

Comment:
```suggestion
Arrange the lines of code below in the following order so that they create a Jest test. You may assume that the test will include correct indentation and closing braces: 
```

Reply:
Thanks for adding the suggestion, sounds great to me!